### DISCUSSION TITLE
Ways to continue beet lyrics from where it was left off

### DISCUSSION THREAD

**Discussion by 2V3EvG4LMJFdRe** [beetbox/beets#3966] [Q&A] [answered]
I've deleted all lyrics from my massive music collection to start anew. With more than a hundred thousand files, `beet lyrics` takes too long to go through every file in a day, or even two. Because of this, I need to find a way not to start from the beginning every time I launch the command. Since the plugin doesn't have a resume function like `import`, the only thing that occurred to me was taking folders with lyrics already fetched outside the library temporarily, and then running `beet lyrics` again with a specified path, but it doesn't work - it either fetches from 0 if I give it the db path, or doesn't fetch at all if I try some arbitrary path.

Any ideas? Some query magic?

**Comment by 2V3EvG4LMJFdRe** [comment, marked as answer]
Ah, it came to me just now. I'll answer myself with `beet lyrics year:`.

**Comment by muness** [comment]
In case someone is interested, I wrote a script to pull lyrics outside of beets on a music folder: https://gist.github.com/muness/539d9c6325ff38e07185bb9e2cfae012 . Not beets specific, but works well with a beets manage folder. It marks folders as processed (by adding a marker file) as it goes, so works well for incremental processing or as you add new folders if the `lyrics` plugin doesn't process them.

**Reply by hlubach** [reply]
Can you tell me how to use it

**Reply by muness** [reply]
> Can you tell me how to use it

I added a README to gist: https://gist.github.com/muness/539d9c6325ff38e07185bb9e2cfae012#file-readme-md . Let me know if you have questions.

**Reply by hlubach** [reply]
Oke I have the script lyrical working, but that create an lrc file  in the music directory|
How to get the lrc files into the mp3 files

**Reply by muness** [reply]
The script does both, saving it into the music file and an lrc. I found different players liked one or the other so I just did both. You can edit the script to remove the part that saves the lrc if you don't want that.
